src/app/shared/services/schema-keys-store.service.ts

```diff
@@ -109,5 +115,24 @@ export class SchemaKeysStoreService {
     return subSchema;
   }
 
+  validateActionsKeypaths(userActions: UserActions): boolean {
```

Comment:
`validateUserActions`

Reply:
does this make sense in this service? fits better in the component or in a new service if it is likely to be reused in the near future.

Reply:
I dont think it's gonna be reused so im moving it to the component where its used